pmaudit

```diff
@@ -524,7 +538,15 @@ def main():
             exclude_set |= {opts.depsfile}
         audit = PMAudit(wdirs, exclude=exclude_set)
         audit.start(flush_host=opts.flush_host, keep_going=opts.keep_going)
-        rc = subprocess.call(cmd)
+        if cmd and cmd[0] and isinstance(cmd[0], list):
+            for cmd_line in cmd: # Execute each line in sequence.
+                rc = subprocess.call(cmd_line)
+                # Stop if we get an error.  Note that "make" does this *even*
```

Comment:
opts.errexit (-e) does have a role, but it's already handled because its "-e" is passed as part of the shell options for running "cmd".  As a result, the "-e" is inserted at the beginning of each line (if "-e" was used), so we don't need to do anything special here.  If "--multiline" wasn't selected, the loop will only happen once (with the complete command); if "--multiline" was selected, we'll loop on each line, and each line will have "-e" if "-e" was a provided option.

Put another way: You can pass in multiple lines of text in the string "cmd", whether or not "--multiline" is enabled.  All "--multiline" does is do what "make" normally does: it runs commands a line at a time, and stops when a line returns an error (nonzero).  Without "--multiline", the shell will get the whole thing at once, and do whatever it does when it gets the whole thing at once.

Here's an example of expected behavior:

~~~~
$ ./pmaudit --multiline -e -c 'echo start; false; echo done 
echo another'
start

$ ./pmaudit --multiline -c 'echo start; false; echo done
echo another'
start
done
another

$ ./pmaudit -e -c 'echo start; false; echo done
echo another'
start

$ ./pmaudit -c 'echo start; false; echo done
echo another'
start
done
another

$ ./pmaudit --multiline -c 'echo start; false
echo another'
start

$ ./pmaudit -c 'echo start; false           
echo another'
start
another
~~~~

Reply:
I used cmds instead of cmd, and modified the README to provide a brief example.